AGENTS.md

```diff
@@ -0,0 +1,38 @@
+# BoardwalkCash Agent Guidelines
+
+## Project Overview
+
+- **Framework**: Remix on Express server
+- **Development**: Nix + devenv, Bun for scripts/tests
+- **Database**: Supabase with migrations in `supabase/migrations`
+
+## Directory Structure
+
+- `app/components/` – Reusable UI components
+- `app/lib/` – Utilities and server helpers
```

Comment:
I think this should be changed. This folder contains non app specific reusable libs